.eslintignore

```diff
@@ -0,0 +1,8 @@
+node-modules/
+actions/notify-workflow-completed/node_modules/
+actions/notify-workflow-completed/dist/
+
+# Following directories have their own formatting configuration:
+actions/npm-version-bump/
+actions/run-workflow/
+actions/upstream-builds-query/
```

Comment:
```suggestion

# Actions have their own linting configuration.
actions/
```

Reply:
We don't have linter configured for `notify-workflow-completed` action, that's why I didn't want to exclude the whole `actions` directory.